flink-libraries/flink-table/src/main/scala/org/apache/flink/table/plan/nodes/datastream/DataStreamOverAggregate.scala

```diff
@@ -112,7 +113,12 @@ class DataStreamOverAggregate(
               "condition.")
         }
       case _: RowTimeType =>
-        throw new TableException("OVER Window of the EventTime type is not currently supported.")
+        if (overWindow.lowerBound.isUnbounded && overWindow.upperBound.isCurrentRow) {
+          createUnboundedAndCurrentRowEventTimeOverWindow(inputDS)
```

Comment:
Updated JIRA [FLINK-5658](https://issues.apache.org/jira/browse/FLINK-5658). @sunjincheng121, not sure if that was an issue with your permissions. It is not possible to modify a closed issue. You need to reopen it first.